Samples/VSTO-shared-code-migration/README.md

```diff
@@ -0,0 +1,169 @@
+
+### Run the VSTO add-in with the shared class library
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. In **Solution Explorer** right-click the **Cell-Analyzer** project and choose **Set as Startup Project**.
+3. From the **Debug** menu, choose **Start Debugging**.
+
+> **Note:** In the completed folder, the Cell Analyzer add-in is already configured to use the shared class library.
+
+## Part 2: Create a REST API wrapper for shared code
+
+The Office web add-in will need to access the shared library. To do this we'll provide a REST API that it can access. We can do this by creating an ASP.NET CORE application that references the class library.
+
+### Explore the REST API project
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. Expand the **CellAnalyzerRESTAPI** project in the solution explorer and open the **/Controllers/AnalyzeUnicodeController.cs** file.
+
+The method **AnalyzeUnicode** is a wrapper that calls into the shared class library to perform the actual operation.
+
+### Run the REST API project
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. In **Solution Explorer** right-click the **CellAnalyzerRESTAPI** project and choose **Set as Startup Project**.
+3. From the **Debug** menu, choose **Start Debugging**.
+4. In the browser that opens, enter the following URL: **https://localhost:44323/api/analyzeunicode?value=test**
+
+You should see the list of characters returned along with each unicode value.
+
+## Part 3: The Office web add-in
+
+The Office web add-in must be created and then modified to match the VSTO add-in UI and document interactions as much as possible. For UI you will need to create similar controls using HTML, CSS, or your favorite library. The UI will look more modern! Also you will need to rewrite code that interacts with the document to use the Office.js API.
+
+### Explore the Office web add-in project
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. Expand the **CellAnalyzer** project in the solution explorer and open the **/Controllers/AnalyzeUnicodeController.cs** file.
+
+The method **AnalyzeUnicode** is a wrapper that calls into the shared class library to perform the actual operation.
+
+### Run the Office web add-in project
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. In **Solution Explorer** right-click the top node **Solution 'Cell-Analyzer'** and choose **Set Startup Projects**.
+3. In the **Solution 'Cell-Analyzer' Property Pages** dialog, select **Multiple startup projects**.
+4. Enable the Start action for each of the following projects.
+    - CellAnalyzerRESTAPI
+    - CellAnalyzerWebAddinWeb
+    - CellAnalyzerWebAddin
+5. Choose **OK**.
+6. From the **Debug** menu, choose **Start Debugging**.
+
+Excel will run and sideload the Office web add-in.
+
+## Security notes
+
```

Comment:
Is this section intentionally empty?

Reply:
Still waiting for security review, then it will have some info. Thanks!